src/modules/linux-micro/fstab/fstab.c

```diff
@@ -42,6 +42,7 @@
 SOL_LOG_INTERNAL_DECLARE_STATIC(_log_domain, "linux-micro-fstab");
```

Comment:
IIRC there are many things in `fstab.c` that is very similar to the filesystem mounting changes in the platform implementation file. Can't them be used here?